common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -48,37 +48,43 @@
     }
 
     public enum Feature {
-        AUTHORIZATION(Type.DEFAULT),
-        ACCOUNT2(Type.DEFAULT),
-        ACCOUNT_API(Type.DEFAULT),
-        ADMIN_FINE_GRAINED_AUTHZ(Type.PREVIEW),
-        ADMIN2(Type.EXPERIMENTAL),
-        DOCKER(Type.DISABLED_BY_DEFAULT),
-        IMPERSONATION(Type.DEFAULT),
-        OPENSHIFT_INTEGRATION(Type.PREVIEW),
-        SCRIPTS(Type.PREVIEW),
-        TOKEN_EXCHANGE(Type.PREVIEW),
-        UPLOAD_SCRIPTS(DEPRECATED),
-        WEB_AUTHN(Type.DEFAULT, Type.PREVIEW),
-        CLIENT_POLICIES(Type.DEFAULT),
-        CIBA(Type.DEFAULT),
-        MAP_STORAGE(Type.EXPERIMENTAL),
-        PAR(Type.DEFAULT),
-        DECLARATIVE_USER_PROFILE(Type.PREVIEW),
-        DYNAMIC_SCOPES(Type.EXPERIMENTAL);
-
+        AUTHORIZATION("Authorization Service", Type.DEFAULT),
```

Comment:
Just a thought: Does it make sense to align the implementation here: https://github.com/keycloak/keycloak/blob/f80d336276fb73603415aedda56b6da4580deb00/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java#L303-L329 with the new Profile.java implementation? Benefit: Having the default value set to enabled/disabled and showing this in the CLI, plus optionally having the same description from the label (if we want this).

Reply:
I agree, but that should be handled by a follow-up PR rather than in this PR.

Reply:
perhaps alaso sth for #9706 ? cc @pedroigor

Reply:
Closing this as it is no longer relevant when we don't have per-feature options